ci/README.md

```diff
@@ -68,15 +85,13 @@ This directory contains the container for CI.
 
 ## steps
 
-This directory contains a few scripts used in CI. Just helps avoid clobbering .travis.yml.
+This directory contains a few scripts used in CI.
+Just helps avoid clobbering .travis.yml.
```

Comment:
I suppose we don't have a `.travis.yml` anymore, maybe we can make this generic with `CI configuration file(s)` or something.